gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -714,6 +715,10 @@ gst_tensor_filter_transform (GstBaseTransform * trans,
     in_tensors[i].size = in_info[i].size - hsize;
   }
 
+  for (i = 0; i < num_tensors; i++) {
```

Comment:
Can you move this after calling invoke callback?
invoke_tensors directly points mapped info (in_info), calling invoke references unmapped memory.

Reply:
Right. Updated.